android/src/ti/map/TiUIMapView.java

```diff
@@ -754,7 +756,9 @@ protected void deselectAnnotation(Object annotation)
 			String title = (String) annotation;
 			TiMarker marker = findMarkerByTitle(title);
 			if (marker != null) {
-				marker.getMarker().hideInfoWindow();
+				if (marker.getMarker() != null) {
+					marker.getMarker().hideInfoWindow();
+				}
```

Comment:
This should also be done for the "showInfoWindow()" call I think.

Reply:
https://github.com/appcelerator-modules/ti.map/blob/ad7ad4dc230b8fe3d84a8b7a8218af50622f04f2/android/src/ti/map/TiUIMapView.java#L739
that one is guarded already